relay-filter/src/interface.rs

```diff
@@ -108,3 +110,41 @@ impl Filterable for Replay {
         self.user_agent()
     }
 }
+
+impl Filterable for Span {
```

Comment:
We might want to extend the impl of the methods returning `None` in case we find a way to access those fields on the `Span`.